doc/testing_policy.md

```diff
@@ -0,0 +1,56 @@
+# Testing policy
+## Contents
+1. [Introduction](#1-introduction)
+2. [How to start Test Suite (Local)](#2-how-to-start-test-suite-local)  
+    2.1 [Using the system build script](#21-using-the-system-build-script)  
+    2.2 [Using standard Go language facilities](#22-using-standard-go-language-facilities)  
+3. [Automated Run Test Suite (Remote)](#3-automated-run-test-suite-remote)  
+
+---
+
+## 1. Introduction
+
+Testing is a very important part of Edge Orchestration project and our team values it highly.
+
+**When adding or changing functionality, the main requirement is to include new tests as part of your contribution.**
+
+We require the Go code to have at **least 70% statement coverage**; please ensure your contributions do not lower the coverage below that minimum. Additional tests are very welcome.
```

Comment:
What about soothing the requirement? (**at least 70% statement coverage**) This is due to my personal concern on pushing a pressure on those who wants to contribute their code to our community, since we are still at the early stage and needs more contributions from 3rd party developers.

Reply:
I suggest to change on:The project team strives to maintain test coverage of at least 70%. We ask you to help us keep this minimum.

Reply:
LGTM about your feedback.

Reply:
Done